libs/agno/agno/workflow/step.py

```diff
@@ -107,6 +107,14 @@ def __init__(
         if step_id is None:
             self.step_id = str(uuid4())
 
+        # Warn if workflow history is enabled for this step
+        if self.add_workflow_history:
+            logger.warning(
+                f"Step '{self.name}' has workflow history enabled (add_workflow_history=True). "
+                "This requires the parent Workflow to have a database configured (e.g., db=SqliteDb()). "
+                "Without a database, workflow history will not persist across runs."
+            )
```

Comment:
So we always log this warning when `add_workflow_history=True`? why do we want this? sounds like the warning on the Workflow level is all we need